gradle-plugin/src/test/groovy/mozilla/appservices/AppServicesExtensionTests.groovy

```diff
@@ -86,40 +86,15 @@ class AppServicesExtensionTests extends Specification {
             """
     }
 
-//    def setup() {
-//        // XXX how to handle this?
-//        SdkHandler.setTestSdkFolder(new File("/Users/nalexander/.mozbuild/android-sdk-macosx"))
-//
-//        testProjectDir.newFolder('src', 'main')
-//        manifestFile = testProjectDir.newFile('src/main/AndroidManifest.xml')
-//        manifestFile << """<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="org.mozilla.appservices.megazord.test"></manifest>"""
-//
-//        project = ProjectBuilder.builder()
-//                .withProjectDir(testProjectDir.root)
-//                .build()
-//
-//        project.pluginManager.apply LibraryPlugin
-//        project.android {
-//            compileSdkVersion 27
-//
-//            defaultConfig {
-//                minSdkVersion 21
-//                targetSdkVersion 27
-//            }
-//        }
-//
-//        project.pluginManager.apply AppServicesPlugin
-//    }
-
     def "megazording a library succeeds"() {
         given:
         androidLibrary()
 
         buildFile << """
             dependencies {
-                // N.b.: there's a version 0.11.5, so this exercises the unit test version inspection.
-                // If it was 0.11.5, taking the latest version might succeed by accident.
-                implementation 'org.mozilla.places:places:0.11.4'
+                // N.b.: there's a version 0.12.2, so this exercises the unit test version inspection.
```

Comment:
I find this comment rather confusing. There _isn't_ a 0.12.2..? Could you reword what this is trying to test? It seems like we're trying to test that requiring a certain version of `org.mozilla.places:places` gets us the right set (and the right versions) of megazords, and none of their ... "children". Unclear what's meant here by the "accident".

Reply:
Yeah, this is awkward, and I will reword.  There will be a 0.12.2 (soon-ish); this was updated from 0.11.{4,5}, and it will still be relevant.  But I will reword.